en/mkdocs.yml

```diff
@@ -358,6 +358,7 @@ nav:
                     - Error Handling: deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/error-handling.md
                     - Enforcer Admin API: deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/enforcer-admin-api/enforcer-v1.0.0.md
                     - Debug Memory Issues: deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md
+                - Updating Choreo Connect: deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md
```

Comment:
SHould be Update Choreo Connect

Reply:
fixed via https://github.com/wso2/docs-apim/pull/6096/commits/ff7abbd3615e65f8560ce14ffc15025f6c9325fc